pipelines/HSC/ApVerify.yaml

```diff
@@ -1,3 +1,4 @@
+# TODO DM-40388: This file will replace ApVerify.yaml, once the new pipeline is vetted.
```

Comment:
Different todo number? And the todo message is a little confusing since this _is_ `ApVerify.yaml`.